### ISSUE TITLE
Problem with patch classifier training (Latest works on Python 3 and TF 2)

### ISSUE THREAD

**Issue by nhhaidee** [lishen/end2end-all-conv#26]
Hi Dr. Li Shen,

I am working on the project for Breast Cancer detection which will be based on your work, however the training for patch classifier produces bad result, loss and accuracy of train is good but the validation is really bad. Please see log file below

The steps what I did 
1. I have 49780 samples for train set and 5580 samples for val set, in which I used TFDS  (https://www.tensorflow.org/datasets/catalog/curated_breast_imaging_ddsm) to generates patches
2. Run 3 stage training as mentioned in the paper
3. Code is implemented by Python 3 and TF 2.10

For now, I don't know what is happening with model training. Can you help me on this? Or give me the hints to fix problem

Thanks,
Hai

```
1556/1556 - 383s - loss: 0.4646 - accuracy: 0.8271 - val_loss: 1.2885 - val_accuracy: 0.6085 - 383s/epoch - 246ms/step
Epoch 9/15
WARNING:tensorflow:Can save best model only with val_acc available, skipping.
WARNING:tensorflow:Can save best model only with val_acc available, skipping.
1556/1556 - 377s - loss: 0.4358 - accuracy: 0.8375 - val_loss: 1.3163 - val_accuracy: 0.6052 - 377s/epoch - 242ms/step
Epoch 10/15
WARNING:tensorflow:Can save best model only with val_acc available, skipping.
WARNING:tensorflow:Can save best model only with val_acc available, skipping.
1556/1556 - 360s - loss: 0.4087 - accuracy: 0.8496 - val_loss: 1.3550 - val_accuracy: 0.6038 - 360s/epoch - 231ms/step
Epoch 11/15
WARNING:tensorflow:Can save best model only with val_acc available, skipping.
WARNING:tensorflow:Can save best model only with val_acc available, skipping.
1556/1556 - 369s - loss: 0.3828 - accuracy: 0.8582 - val_loss: 1.3358 - val_accuracy: 0.6178 - 369s/epoch - 237ms/step
Epoch 11: early stopping
```

**Comment by nhhaidee** [comment]
@lishen When you have time, can you help me on this issue?
